migrations/eap8.0/eap7.4/src/main/java/org/jboss/migration/eap7/to/eap7/EAPXP7_4ToEAPXP8_0ServerMigrationProvider.java

```diff
@@ -18,8 +18,10 @@
 
 import org.jboss.migration.eap.EAPXPServer7_4;
 import org.jboss.migration.eap.EAPXPServerMigrationProvider8_0;
-import org.jboss.migration.eap.task.hostexclude.EAP8_0AddHostExcludes;
+import org.jboss.migration.eap.task.hostexclude.EAPXP8_0AddHostExcludes;
```

Comment:
AFAIK, domain mode is not supported in EAP XP.

Reply:
the tool simply syncs host excludes config to what will be used on XP5, which is diff then EAP 8